cmd/api/main.go

```diff
@@ -149,6 +150,7 @@ func cmd() *cobra.Command {
 	c.Flags().BoolVar(&opts.enableFeatureBindings, "enable-feature-bindings", false, "Enables the Endpoint Bindings controller")
 	c.Flags().StringSliceVar(&opts.bindings.allowedURLs, "bindings-allowed-urls", []string{"*"}, "Allowed URLs for Endpoint Bindings")
 	c.Flags().StringVar(&opts.bindings.name, "bindings-name", "default", "Name of the Endpoint Binding Configuration")
+	c.Flags().StringVar(&opts.bindings.ingressEndpoint, "bindings-ingress-endpoint", "", "The endpoint the bindings forwarder connects to")
```

Comment:
So I think this field is actually returned server side from the k8sop registration request, see: https://github.com/ngrok-private/ngrok/blob/main/proto/api/kubernetes_operator.proto#L214

Reply:
Talked offline: It is, but we don't have defaulting for it yet in the API and there may be the case that we need to whitelabel and use a custom domain, so we'll leave it for now. Once the API returns this and supports custom overrides, we'll drop this. Probably also the agent `serverAddr` at the same time as well.